### PR TITLE
Upgrade to Pytorch 1.4.

### PR DESCRIPTION
**Pull request by stephenroller**
**Patch description**
With the [release of pytorch 1.4](https://pytorch.org/blog/pytorch-1-dot-4-released-and-domain-libraries-updated/), we officially remove support for pytorch 1.2 and only support 1.3 and 1.4.

**Testing steps**
CI.

### CODE REVIEW

**Comment by stephenroller** [comment]
The mturk flakiness will be addressed in the next PR.

File: tests/nightly/gpu/test_wizard.py [hunk 1]
```diff
@@ -53,13 +53,13 @@ def setUpClass(cls):
         display_data.display_data(opt)
 
     def test_end2end(self):
-        stdout, valid, _ = testing_utils.eval_model(END2END_OPTIONS)
```
Comment (stephenroller) [inline tests/nightly/gpu/test_wizard.py]: these are bits missed from #2336.

File: tests/test_image_seq2seq.py [hunk 2]
```diff
@@ -61,10 +53,7 @@
 }
 
 
-@unittest.skip(
-    "need pytorch 1.4 release, https://github.com/pytorch/vision/issues/1712"
-)
-@unittest.skipUnless(TORCH_AVAILABLE, 'Must use torch 1.2 or above')
+@testing_utils.skipUnlessTorch14
```
Comment (EricMichaelSmith) [inline tests/test_image_seq2seq.py]: What does Torch 1.4 do that allows us to run these tests now?

Comment (stephenroller) [reply tests/test_image_seq2seq.py]: There was a bug in torch vision.

**Pull request merged**